README.md

```diff
@@ -1,31 +1,118 @@
 # Open-Catalyst-Dataset
-Workflow for creating and analyzing the Open Catalyst Dataset.
 
-We choose a bulk, and from that bulk we can choose from different surfaces. Separately, we choose an adsorbate. Then, the adsorbate is placed on the surface with multiple placement options. VASP input files are generated for the adsorbate placed on the surface, as well as the surface alone.
+This repository hosts the input generation workflow used in the Open Catalyst Project.
 
+## Setup
 
-# Dependencies
-See `setup.py`
+The easiest way to install prerequisites is via [conda](https://conda.io/docs/index.html). After installing [conda](http://conda.pydata.org/), run the following commands: 
 
-# Usage
+* Create a new environment, `vaspenv`, `conda create -n vaspenv python=3.7`
+* Activate the newly created environment to install the required dependencies: `conda activate vaspenv`
+* Install specific versions of Pymatgen and ASE: `pip install pymatgen==2020.4.2 ase>=3.19.1`
+* Install Catkit from Github: `pip install git+https://github.com/SUNCAT-Center/CatKit.git#egg=catkit catkit`
+
```

Comment:
Can we keep `setup.py` and streamline it to something like this: https://github.com/Open-Catalyst-Project/ocp/blob/2483fbb930f376c546c2e4174851f656a77bbbfc/setup.py#L1-L17.  Also adding an additional step here: Install this package: `pip install -e .` I've been using this to import things from this repo `import ocdata.blahblah` for instance.

Reply:
For CatKit, could we use the forked one in Ulissigroup? https://github.com/ulissigroup/CatKit?

Reply:
@mshuaibii I put `setup.py` back in but not sure what you meant by streamlining it - am I supposed to shorten it to match what you have in ocp (what I currently have)? And where am I adding the `pip install -e`?

Reply:
Sorry missed this - yup this looks good! `pip install -e` is just another step. So after that last installation, `cd this_directory`, `pip install -e .`